src/sonic-config-engine/minigraph.py

```diff
@@ -285,18 +285,33 @@ def get_mgmt_info(devices, dev, port):
 
     return ret_val
 
-def get_alias_map_list(hwsku):
-    alias_map_json = os.path.join('/usr/share/sonic', hwsku, 'alias_map.json')
-    if not os.path.isfile(alias_map_json):
+def get_alias_map_list(hwsku, platform=None):
+    port_config_candidates = []
+    port_config_candidates.append('/usr/share/sonic/sku/port_config.ini')
+    if platform != None:
+        port_config_candidates.append(os.path.join('/usr/share/sonic/device', platform, hwsku, 'port_config.ini'))
+    port_config_candidates.append(os.path.join('/usr/share/sonic/device', hwsku, 'port_config.ini'))
```

Comment:
For supporting usage in both base os and docker container, and for backward compatibility

---
In reply to: [101695827](https://github.com/Azure/sonic-buildimage/pull/304#discussion_r101695827) [](ancestors = 101695827)